pkg/apb/ext_creds_test.go

```diff
@@ -7,197 +7,23 @@ import (
 	ft "github.com/openshift/ansible-service-broker/pkg/fusortest"
 )
 
-func TestDecodeOutput(t *testing.T) {
-	output := []byte(`
```

Comment:
Why did we take out this test? I was verifying decodeOutput directly. That way we can test any of the error conditions if any.